datafusion/core/tests/sql/expr.rs

```diff
@@ -721,6 +721,14 @@ async fn test_like() -> Result<()> {
     Ok(())
 }
 
+#[tokio::test]
+async fn test_unicode_escaped() -> Result<()> {
+    test_expression!("U&'pppp'", "ppp");
```

Comment:
This test looks strange (4 ps compared with 3). I guess the error will surface once the PR updates to newer sqlparser rev. Approving regardless.

Reply:
fixed